src/resource/resource.hpp

```diff
@@ -102,6 +105,10 @@ class Model {
     friend bool operator==(const Model& lhs, const Model& rhs);
 };
 
+}  // namespace cppsdk
+}  // namespace viam
+
+using namespace viam::cppsdk;
```

Comment:
No, you can't do `using namespace viam::cppsdk` in a header, because it defeats the namespacing for consumers (direct or indirect) of that header. Instead, I'd expect you can just fully qualify our types:

```
template<>
struct std::hash<::viam::cppsdk::Name> {
...
```

Please give that a try and LMK if it doesn't work out as you expect for some reason.

Reply:
That works! Change made.